modules/metering-about-install.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-installing-metering.adoc
+
+[id="metering-about-install_{context}"]
+= About Installing Metering
+
+Metering leverages a SQL database called Presto.
+Presto itself doesn't actually store the data itself, instead it's storage is decoupled from it, often in the form of object storage.
+This means at a minimum, you need to configure persistent storage to hold reporting data, and the Hive metastore to hold metadata about database tables managed by Presto and Hive.
```

Comment:
grammatically, it should be `...hole reporting data and the Hive` (w/o the comma since use `,` only for list of 3 or more, @lamek is that right?

Reply:
I guess I tend to use an oxford comma for `and` conjunctions.

Reply:
Actually, I didn't write this @lamek did, this I just pulled out into this separate module from what he had in the install doc.

Reply:
ok, then I guess it's ok then...